app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
     <!-- NOTE TO DEVELOPER: You can enable debug logs for fast upload & better inspection in Firebase console using https://support.google.com/analytics/answer/7201382. -->
     <meta-data
       android:name="firebase_analytics_collection_deactivated"
-      android:value="false" />
+      android:value="true" />
```

Comment:
Just to check ... this is correct?

Also the expiration date a few lines below says "2020-09-01" ... is that correct? Seems old.